packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -103,7 +103,8 @@ class AndroidWorkflow extends DoctorValidator implements Workflow {
         ));
       } else {
         messages.add(new ValidationMessage.error(
-          'Android SDK not found. Download from https://developer.android.com/sdk/\n'
+          'Android SDK not found. Download Android Studio and run it to complete\n'
```

Comment:
Maybe "Install Android Studio from https://developer.android.com/studio/index.html. On first launch it will assist you in installing the Android SDK components"?

In any case, it's definitely better than before.  rslgm.